.env.example

```
@@ -1,95 +1,200 @@
+# Basic Crawling Settings
 CRAWL_HEADLESS=true
 CRAWL_BROWSER=chromium
-CRAWL_MAX_PAGES=100
+CRAWL_MAX_PAGES=1000
 CRAWL_MAX_DEPTH=3
-MAX_CONCURRENT_CRAWLS=25
-CRAWLER_DELAY=0.1
-CRAWLER_TIMEOUT=15.0
-CRAWL_MIN_WORDS=100
+MAX_CONCURRENT_CRAWLS=10
+CRAWLER_DELAY=0.05
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CRAWLER_DELAY key should go before the CRAWL_BROWSER key

(UnorderedKey)